planning/behavior_path_planner_common/include/behavior_path_planner_common/interface/scene_module_interface.hpp

```diff
@@ -460,9 +457,9 @@ class SceneModuleInterface
   virtual bool canTransitFailureState() = 0;
 
   /**
-   * @brief State transition condition IDLE -> RUNNING
+   * @brief Explicitly set the initial state
    */
-  virtual bool canTransitIdleToRunningState() = 0;
+  virtual ModuleStatus setInitState() { return ModuleStatus::RUNNING; }
```

Comment:
Can this method be `const` ?

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/6051/commits/0c7293999ddacc92b062298a1d371dfaab3f79ae